doc/version_history.rst

```diff
@@ -9,6 +9,11 @@ Version History
 v0.9.4
 ======
 
+* Update the versions of several hooks.
+
+v0.9.4
```

Comment:
Remove the duplicated v0.9.4?

Reply:
Good catch. It was meant to be v0.9.5 and I forgot to update it.